examples/lighting-app/nrfconnect/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,39 @@
+/*
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/ {
+	/*
+	* By default, PWM module is only configured for led0 (LED1 on the board).
+	* The lighting-app, however, uses LED2 to show the state of the lighting,
+	* including its brightness level.
+	*/
+	aliases {
+		pwm-led1 = &pwm_led1;
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_led1: pwm_led_1 {
+			pwms = < &pwm0 0x1d >;
+		};
+	};
+
+};
+
+&pwm0 {
```

Comment:
Please align with the NCS light_bulb. I needed to add some more things to make sure that PWM doesn't affect LED1.